src/context/context.ts

```diff
@@ -71,8 +71,8 @@ export class Context {
   public getSync (paths: PropertyKey[]): unknown {
     return toValueSync(this._get(paths))
   }
-  public * _get (paths: PropertyKey[]): IterableIterator<unknown> {
-    const scope = this.findScope(paths[0])
+  public * _get (paths: (PropertyKey | Drop)[]): IterableIterator<unknown> {
```

Comment:
I think we need change the type of `PropertyKey` to include `Drop`

Reply:
I thought of that, but the issue is it used in some other places where a Drop wouldn't make sense. Also the fact that it collides with the builtin type (type PropertyKey = string | number | symbol) makes it a bit confusing sometimes.